spotify

```diff
@@ -188,8 +188,7 @@ while [ $# -gt 0 ]; do
 
         "vol"    )
             vol=`osascript -e 'tell application "Spotify" to sound volume as integer'`;
-            text=$2;
-            if [[ "$2" = "show" || "$2" = "" ]]; then
```

Comment:
I say we do that. (As in just change the docs to now default to `vol`, and leave the old `vol show` behaviour as is and undocumented.) This way, we get the new behaviour we want, *and* don't break any backward compatibility with anyone using or scripting around `vol show`.